File: Beginner/helpChef/helpChef.c
#include <stdio.h>

int main(void){
	int T,N;

	while(1){
		scanf("%d",&T);

		if(T<=0 || T>1000){
			printf("T must be 1 <= T <= 1000\n");
		}else{
			break;
		}
	}

	while(T>0){

		while(1){
			scanf("%d",&N);

			if(N<-20 || N>20){
				printf("N must be -20 <= N <= 20\n");
			}else{
				break;
			}
		}

		if(N<10){
			printf("Thanks for helping Chef!\n");
		}else{
			printf("-1\n");
		}

		T--;
	}

	return 0;
}

File: Beginner/helpChef/helpChef.java
import java.util.Scanner;

public class helpChef{
	public static void main(String[] args) {
		int T,N;
		Scanner scan = new Scanner(System.in);

		while(true){
			T = scan.nextInt();

			if(T<=0 || T>1000){
				System.out.println("T must be 1 <= T <= 1000");
			}else{
				break;
			}
		}

		while(T>0){

			while(true){
				N = scan.nextInt();

				if(N<-20 || N>20){
					System.out.println("N must be -20 <= N <= 20");
				}else{
					break;
				}
			}

			if(N<10){
				System.out.println("Thanks for helping Chef!");
			}else{
				System.out.println("-1");
			}

			T--;
		}
	}
}